README.md

```diff
@@ -70,3 +70,21 @@ All lead associations will be based on the domain of the experience being viewed
 <script id="ceros-marketo-munchkin-plugin" src="//sdk.ceros.com/marketo-munchkin/main-v0.js" accountId="AAA-111-BBB"></script>
 ```
 2. Replace the value of the accountId attribute with your marketo munchkin Id
+
+## The State of the Layer Plugin
+
+Allows people using the Ceros Studio's SDK pallet to tag components to enable them to both control, and be controlled by, parameters in the Experience's URL. It can be used to remember user's actions between pages or to trigger actions like showing layers or jumping to anchor points when the Experience is opened by the user.
```

Comment:
I mean like, I am confused about what this is actually doing, and is there a different way of accomplishing what we want that's already built in to Ceros. 

Just looking for a use case to help me understand.